bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/TitleImpl.java

```diff
@@ -79,7 +79,12 @@ private void initModel() {
         if (request != null && i18n == null) {
             i18n = GuideUtils.getI18n(request, resource);
         }
-
+        if (StringUtils.isBlank(title)) {
```

Comment:
how are ensuring that the title on saved on forms container when editing title component?